app/routes/admin/users/index.js

```diff
@@ -5,91 +5,8 @@ const { Route } = Ember;
 export default Route.extend({
```

Comment:
Please do not make any API calls here remove the model & redirect to list route.